irohad/consensus/yac/transport/yac_pb_converters.hpp

```diff
@@ -56,14 +56,14 @@ namespace iroha {
           if (vote.hash.block_signature) {
             auto block_signature =
                 pb_vote.mutable_hash()->mutable_block_signature();
-            auto const &signature = iroha::hexstringToBytestring(
+            auto signature = hexstringToBytestringResult(
                 vote.hash.block_signature->signedData());
-            assert(signature);
-            auto const &public_key = iroha::hexstringToBytestring(
+            assert(expected::hasValue(signature));
```

Comment:
after #348 is merged, a failure in `assumeValue` (below) will provide more information (`iroha::to_string(<error>)`) than just `assert(hasValue(<>)`). So I'd remove such assertions.